### PR TITLE
save retry records for ES pillow errors

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?222981
@NoahCarnahan @gcapalbo 

buddy @dannyroberts

### CODE REVIEW

File: corehq/couchapps/domain_shows/shows/domain_date.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+function (doc, req) {
```
Comment (dimagimon) [inline corehq/couchapps/domain_shows/shows/domain_date.js]: Parsing error: Unexpected token (

**Comment by czue** [comment]
dang, that's unfortunate. :+1: to merge from me, but just wanted to confirm we're sure this doesn't need a reindex? I think it doesn't i'm just being paranoid.

**Comment by snopoke** [comment]
I'm pretty sure it doesn't but can verify tomorrow

**Comment by snopoke** [comment]
Verified that this won't cause a re-index. Here's a good article explaining how to verify: http://www.staticshin.com/programming/does-updating-a-design-document-in-couchdb-cause-rebuilding-of-views/

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**